pkg/infra/config_test.go

```diff
@@ -38,8 +35,8 @@ args:
   - b
   - 1
 mspid: Org1MSP
-private_key: /path/to/private.key
-sign_cert: /path/to/sign.cert
+private_key: ../../test/private.key
+sign_cert: ../../test/sign.cert
```

Comment:
for UTs like this, we should generate certs/keys file on demand (with tempFile and tempDir) and remove them afterwards, obviously some random bytes would be enough here